server/server.go

```diff
@@ -7,17 +7,21 @@ import (
 	"os"
 	"time"
 
+	"github.com/google/uuid"
 	proxy "github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/spacemeshos/smutil/log"
+	"go.uber.org/zap"
 	"golang.org/x/net/context"
```

Comment:
```suggestion
	"context"
```

This isn't the only place in poet where the outdated package is used. Imo we should use `context` instead (`golang.org/x/net/context` today is just an alias to `context`)

Reply:
Do you know if there is a linter to detect such problems?

Reply:
Yes, there is! `depguard` can be used for this and it is part of `golangci-lint`. See this PR https://github.com/spacemeshos/go-spacemesh/pull/3800. In it I enable it and fix some additional findings it reported.